libs/barista-components/sunburst-chart/src/sunburst-chart.ts

```diff
@@ -194,110 +235,47 @@ export class DtSunburstChart {
       this._selectedRelativeValue = 1;
     }
   }
-  /** Programmatically close the overlay. */
-  closeOverlay(): void {
-    this._dismissOverlay();
-  }
 
-  /** Puts the overlay in place. */
-  openOverlay(node: DtSunburstChartSlice): void {
-    const origin = this._getOriginFromSlice(node);
-
-    this._dismissOverlay();
-    if (origin) {
-      //  TODO: lukas.holzer
-      // Bazel cannot resolve the extends of the interface properly and
-      // throws error: Property 'data' does not exist on type 'DtSunburstChartSlice'.
-      // @ts-ignore
-      this._createOverlay(origin, node.data);
-    }
-  }
```

Comment:
If you deprecate the code, you cannot actually delete it. It still needs to work. 
Why are you removing this functionality exactly? Nothing was stated in the issue that you are planning to remove this.